src/test/java/com/snowflake/kafka/connector/internal/SinkServiceIT.java

```diff
@@ -749,19 +579,23 @@ public void testNativeNullIngestion() throws Exception {
     service.closeAll();
   }
 
-  @Test(expected = SnowflakeKafkaConnectorException.class)
```

Comment:
We need to allow null value ingestion because community converters create null values instead of empty SnowflakeRecordContents. Currently Snowpipe fails to ingest null value records with community converters.